main.tex

```diff
@@ -306,35 +269,45 @@ \subsection*{\textit{Vgsc} non-synonymous nucleotide variation}
 %%
 
 
+%% Figure - Pairwise LD.
+%
+\begin{figure}[!t]
+
+  \centering
+  
+  \includegraphics[width=\linewidth]{artwork/fig_ld.png}
+  
+  \caption{\textbf{Linkage disequilibrium between non-synonymous variants}. A value of 1 indicates that the two alleles are in perfect linkage, meaning that one of the two alleles is only ever found in combination with the other. Conversely, a value of -1 indicates that the two alleles are never found in combination with each other.}
+  
+  \label{fig:ld}
+ 
+\end{figure}
```

Comment:
I moved the figure down because it was occurring on the next page after table 1 and so thought better to spread out the display items.